crates/astria-conductor/src/conductor.rs

```diff
@@ -99,12 +95,26 @@ impl Conductor {
 
         let signals = spawn_signal_handler();
 
+        let rollup_id =
+            proto::native::sequencer::v1alpha1::RollupId::from_unhashed_bytes(&cfg.chain_id);
```

Comment:
rename the config value also to `rollup_id`

Reply:
I wasn't sure about this. It's m technically not the ID but just the name. The ID are the 32 bytes and I don't want to cause confusion between these two.

Is rollup_name good? Also CC @joroshiba